src/dali/transaction_resolver.py

```diff
@@ -127,12 +127,13 @@ def _update_spot_price_from_web(transaction: AbstractTransaction, historical_pri
             LOGGER.debug("Reading spot_price for %s/%s from cache: %s", key.timestamp, key.asset, spot_price)
         else:
             time_granularity: List[int] = [60, 300, 900, 3600, 21600, 84600]
-            from_timestamp: str = transaction.timestamp_value.strftime("%Y-%m-%d-%H-%M")
+            transaction_utc_timestamp = transaction.timestamp_value.astimezone(timezone.utc)  # Coinbase API expects UTC timestamps only
```

Comment:
Could you add the link you found (https://forums.coinbasecloud.dev/t/invalid-end-on-product-candles-endpoint/320) on this in the comment?